### ISSUE TITLE
Undefined Variable after Generating Block

### ISSUE THREAD

**Issue by chaseconey** [RicLeP/laravel-storyblok#6]
Looks like a recent change to the generator command stub changed the variable in the blade file from `$content` to `$block`. Now when generating a fresh block with the latest version of the library, you get this error:

```
Undefined variable: block (View: /projects/www.bankrate.com/resources/views/storyblok/blocks/teaser.blade.php)
```

**Comment by RicLeP** [comment]
Hi, I’m going to need to think of a fix for this, it was caused by another bug - the `Block->render()` method used the variable `block` when passing the data to the view which main it incompatible with the generator.

Are you doing something like this in your blade? I will need to update the docs too to keep things consistent.
`@include('storyblok.blocks._' . $section->component(), ['content' => $section])`

I will look at adding custom stubs, probably in `resources/stubs/storyblok` of your application and use those if they are found. Would that work for you?

**Comment by RicLeP** [comment]
I’ve released 2.5.5 which allows you to supply custom stubs for the generator. See the docs here: https://ls.sirric.co.uk/docs/2.5/blocks#artisan-commands

I think this should solve your problem, let me know.
